src/components/component_styles/evseLiveDisplay.css

```diff
@@ -29,16 +29,17 @@
     padding:10px;
     background:rgba(255, 255, 255, 0.6);
     text-decoration: none;
+    white-space: nowrap;
+    flex: 1 0 18%;
+    min-width: 100px;
+    max-width: 200px;
```

Comment:
If you want to manage the layout of the child-columns, it is cleaner to manage that in the parent component:
Try `grid-template-columns: repeat(3, minmax(100px, 1fr));` in `.evse-attr`. I think it should have the same effect.